serverless_llm/serve/routers/roundrobin_router.py

```diff
@@ -137,6 +137,38 @@ async def generate(self, request_data: dict):
             self.request_count -= 1
         return result
 
+    async def encode(self, request_data: dict):
```

Comment:
This function only has little difference with `generate`. Try to figure out an extendable design to refuse `generate` function.

Reply:
I have resolved it in the new commit